openmc/deplete/abc.py

```diff
@@ -634,10 +647,27 @@ class Integrator(ABC):
         Size of each depletion interval in [s]
     power : iterable of float
         Power of the reactor in [W] for each interval in :attr:`timesteps`
+    solver : callable
+        Function that will solve the Bateman equations
+        :math:`\vec{n}_{i+1} = A_i\vec{n}_i` with a step size :math:`t_i`. Can
```

Comment:
Should that be n_{i+1} = exp(A_i t_i) n_i?

Reply:
Yes. I think I was trying to make the ODE representation and left of the `d/dt`. Will update